playwright_test/locaterPackage/kmLocators.js

```
@@ -0,0 +1,71 @@
+    kmSmirkEmoji : '//*[@id="group_0"]/a[6]',
+    kmLocSendBtn : '#mck-loc-submit',
+    kmQuickRepliesBtn : '#intent-option',
+    kmQuickReplyOptions : '//li[text()="Book a Demo"]'
+};
+
+export const RICHMESSAGES_LOCATORS = {
+    kmSuggestedRepliesYesBtn : '//button[@aria-label="Yes"]',
+    kmSuggestedRepliesYesBtnResponse : '//div[text()="Cool! send me more."]',
+    kmLinkButton : '//button[@aria-label="Go To Google"]',
+    kmLinkOnNewTab : '//img[@alt="Google"]',
+    kmImageResponse: '//div[@class="km-image-template"]',
```

Biome (1.9.4)

[error] This property value named kmImageResponse is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named kmImageResponse


(lint/suspicious/noDuplicateObjectKeys)